src/blog/2024/09/how-to-debug-node-red-flows.md

```diff
@@ -0,0 +1,174 @@
+---
```

Comment:
Why is this a blog post and not a page  under node-red?

Reply:
i will move it under the Node-RED Page